crypto/s2n_rsa_signing.h

```diff
@@ -18,9 +18,20 @@
 #include <s2n.h>
 
 #include "utils/s2n_blob.h"
+#include "crypto/s2n_openssl.h"
+#include "crypto/s2n_rsa.h"
+
+/* RSA PSS Signing is supported in OpenSSL 1.0.2 and above */
+#if S2N_OPENSSL_VERSION_AT_LEAST(1, 1, 1) && !defined(LIBRESSL_VERSION_NUMBER)
+#define RSA_PSS_SUPPORTED 1
```

Comment:
Can we make this (and the is_supported function) more specific? "rsa_pss_signing" or "rsa_pss_rsae"? RSA_PSS could refer to either the certs or the signing.